api/youtube_downloader.py

```
@@ -4,10 +4,15 @@
 import xml.etree.ElementTree as ET
 from datetime import timedelta, datetime
 import logging
+import os
 from typing import Dict, List, Optional, Union
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)